VCTwoFluidStaggeredStokesOperator.cpp

```diff
@@ -325,6 +391,24 @@ VCTwoFluidStaggeredStokesOperator::apply(SAMRAIVectorReal<NDIM, double>& x, SAMR
     StaggeredStokesPhysicalBoundaryHelper::resetBcCoefObjects(d_un_bc_coefs, d_P_bc_coef);
     d_hier_bdry_fill->resetTransactionComponents(d_transaction_comps);
 
+#define USE_DIV
+#define USE_SYNCHED_INTERP
+#define POST_SYNCH

```

Comment:
In particular, these three lines control which changes are used:

- `USE_DIV` synchronizes the side centered values before computing the divergence of the velocities.
- `USE_SYNCHED_INTERP` uses a synchronized node and side interpolated version of the volume fraction
- `POST_SYNCH` performs an extra synchronization step at the end of the `apply()` function.

You can switch back to the old behavior by commenting out one of these options and rebuilding.